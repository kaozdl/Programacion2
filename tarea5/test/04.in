# Caso 04
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
reiniciar_sistema
imprimir_totales
imprimir_totales
imprimir_totales
imprimir_totales
imprimir_totales
imprimir_totales
imprimir_totales
imprimir_totales
imprimir_totales
procesar_rango 1
procesar_rango 2
procesar_rango 3
procesar_rango 4
procesar_rango 5
procesar_rango 1
procesar_rango 2
procesar_rango 3
procesar_rango 4
procesar_rango 5
procesar_rango 1
procesar_rango 2
procesar_rango 3
procesar_rango 4
procesar_rango 5
Fin

