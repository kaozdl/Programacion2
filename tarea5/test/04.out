1>2>
  REINICIO DE SISTEMA 
3>
  REINICIO DE SISTEMA 
4>
  REINICIO DE SISTEMA 
5>
  REINICIO DE SISTEMA 
6>
  REINICIO DE SISTEMA 
7>
  REINICIO DE SISTEMA 
8>
  REINICIO DE SISTEMA 
9>
  REINICIO DE SISTEMA 
10>
  REINICIO DE SISTEMA 
11>
  REINICIO DE SISTEMA 
12>
  REINICIO DE SISTEMA 
13>
  REINICIO DE SISTEMA 
14>
  REINICIO DE SISTEMA 
15>INFORME POR RANGOS
0
0
0
0
0
16>INFORME POR RANGOS
0
0
0
0
0
17>INFORME POR RANGOS
0
0
0
0
0
18>INFORME POR RANGOS
0
0
0
0
0
19>INFORME POR RANGOS
0
0
0
0
0
20>INFORME POR RANGOS
0
0
0
0
0
21>INFORME POR RANGOS
0
0
0
0
0
22>INFORME POR RANGOS
0
0
0
0
0
23>INFORME POR RANGOS
0
0
0
0
0
24>25>26>27>28>29>30>31>32>33>34>35>36>37>38>39>Fin.
